modules/ROOT/pages/logs-using.adoc

```diff
@@ -34,21 +37,18 @@ You can create a filter to search logs using predefined fields, operators, and v
 
 . Click *+ Add a filter*. 
```

Comment:
*Add filter*

Reply:
It's actually +Add a filter in the UI (not what's in the image). But I fixed the space issue.